raft/log.go

```diff
@@ -586,289 +624,343 @@ func (l *Log) Leave() error {
 	return nil
 }
 
-// setState moves the log to a given state.
-func (l *Log) setState(state State) {
-	l.Logger.Printf("log state change: %s => %s", l.state, state)
+// startStateLoop begins the state loop in a separate goroutine.
+// Returns once the state has transitioned from "stopped".
```

Comment:
Is this comment correct? Surely it returns once the state of the node reaches the state passed into the function?